action.yaml

```
@@ -0,0 +1,15 @@
+name: 'Run ESLint'
+description: 'Run ESLint'
+inputs:
+  repo-token:
+    description: 'Access to your github repository. Should be secrets.GITHUB_TOKEN '
+    required: true
+  source-root:
+    description: 'change into the given sub dir to perform this scan.'
+    required: false
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+  args:
+  - "${{ inputs.repo-token }}"
+  - "${{ inputs.source-root }}"
```

I'd prefer the metadata stay the same wherever it's possible to match my existing `Dockerfile`

```suggestion
name: ESLint Action
description: Lint your Javascript projects with inline lint error annotations on pull requests.
branding:
  icon: code
  color: yellow
inputs:
  repo-token:
    description: GitHub repository token for publishing inline lint annotations. Should be secrets.GITHUB_TOKEN.
    required: true
  source-root:
    description: Root directory to lint files in.
    required: false
runs:
  using: docker
  image: Dockerfile
  args:
  - "${{ inputs.repo-token }}"
  - "${{ inputs.source-root }}"
```